src/applications/hca/config/chapters/insuranceInformation/general.js

```diff
@@ -1,73 +1,61 @@
-import merge from 'lodash/merge';
-import get from 'platform/utilities/data/get';
+import React from 'react';
 import fullSchemaHca from 'vets-json-schema/dist/10-10EZ-schema.json';
+import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
 
-import InsuranceProviderView from '../../../components/InsuranceProviderView';
-
-const { provider } = fullSchemaHca.definitions;
 const { isCoveredByHealthInsurance } = fullSchemaHca.properties;
 
+const insuranceInfo = (
+  <p>
+    <AdditionalInfo triggerText="Why we ask for this information">
```

Comment:
I think this will show some render errors... like a `<div>` contained within a `<p>` is invalid markup.

Reply:
okay. thx fixed.